### PR TITLE
Use revision, not version to indicate revision on flow steps API

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

**Comment by nicpottier** [comment]
plz review.

**Comment by ericnewcomer** [comment]
Looks good. Maybe leave a test using version to ensure backwards compat?

**Comment by nicpottier** [comment]
I did.

File: temba/api/tests.py [hunk 1]
```diff
@@ -713,18 +713,25 @@ def test_api_steps(self):
...
+            self.assertEquals('Invalid revision: 12', response.json['steps'][0])
 
             # this one exists and has our node
+            data['revision'] = 2
+            response = self.postJSON(url, data)
+            self.assertEquals(201, response.status_code)
+            self.assertIsNotNone(self.joe.urns.filter(path='+13605551212').first())
+
+            # test with old name
+            del data['revision']
```
Comment (nicpottier) [inline temba/api/tests.py]: over here.

Comment (ericnewcomer) [inline temba/api/tests.py]: oh, missed that. ha.

**Pull request merged**